website/docs/guides/core-to-cloud-3.md

```diff
@@ -0,0 +1,192 @@
+- [Rerun your jobs](/docs/deploy/retry-jobs) from the start or the point of failure if your dbt job run completed with a status of **`Error.`**
+
+### Caveats
+- To automate the setup and configuration of your dbt Cloud platform, you can store your job configurations as code within a repository:
+  - Check out our [Terraform provider.](https://registry.terraform.io/providers/dbt-labs/dbtcloud/latest/docs/resources/job)
+  - Alternatively, check out our [jobs-as-code](https://github.com/dbt-labs/dbt-jobs-as-code) repository, which is a tool built to handle dbt Cloud jobs as a well-defined YAML file.
+- dbt Cloud users and external emails can receive notifications if a job fails, succeeds, or is cancelled. To get notifications for warnings, you can create a [webhook subscription](/guides/zapier-slack) and post to Slack.
+
+## dbt Mesh
+
+[dbt Mesh](/best-practices/how-we-mesh/mesh-1-intro) helps organizations with mature, complex transformation workflows in dbt increase the flexibility and performance of their dbt projects. It deploys multiple interconnected dbt projects instead of a single large, monolithic project.
+
+It enables you to integrate and navigate between different projects and models with [cross-project dependencies](/docs/collaborate/govern/project-dependencies#how-to-write-cross-project-ref), enhancing collaboration and data governance.
+
+Here are some tips and caveats to consider when using dbt Mesh:
+
+### Tips
+- To use [cross-project references](/docs/collaborate/govern/project-dependencies#how-to-write-cross-project-ref), all developers need to develop with dbt Cloud (either with the dbt Cloud CLI or dbt Cloud IDE). Cross-project references are not supported in dbt Core.
+- Link models across projects for a modular and scalable approach for your project and teams.
+- Manage access to your dbt models both within and across projects using:
+  - **[Groups](/docs/collaborate/govern/model-access#groups)** &mdash; Organize nodes in your dbt DAG that share a logical connection and assign an owner to the entire group.
+  - **[Model Access](/docs/collaborate/govern/model-access#access-modifiers)** &mdash; Control who can reference models.
+  - **[Model Versions](/docs/collaborate/govern/model-versions)** &mdash; Enable adoption and deprecation of models as they evolve.
+  - **[Model Contracts](/docs/collaborate/govern/model-contracts)** &mdash; Set clear expectations on the shape of the data to ensure data changes upstream of dbt or within a project's logic don't break downstream consumers' data products.
+- Use [dbt-meshify](https://github.com/dbt-labs/dbt-meshify) to separate your mono-repo and move to dbt Cloud.
+
+### Caveat
+- Currently, dbt Mesh does not support circular project dependencies.
```

Comment:
@jtcohen6 i think you mentioned you had some more bits to add here for mesh?

Reply:
@mirnawong1 Here are the top three I would list. Probably too much text, so feel free to cut it down.

### Caveats

**Use of cross-project references requires that each upstream dbt project maps to exactly one dbt Cloud project.** We strongly discourage defining multiple projects for the same codebase, whether for the purposes of managing permissions, data warehouse connections, and isolation of production from non-production data. While this was required historically, this should no longer be necessary with the rollout of Staging environments (beta, link), Environment-level RBAC (coming soon), and Extended Attributes (link).

**Project dependencies are uni-directional.** dbt will check for cycles across projects and raise errors if any are detected. We are considering support for bi-directional project dependencies in the future, whereby dbt would still check for node-level cycles while allowing cycles at the project level.

**Public model metadata is visible to everyone in the account.** The goal is to improve discoverability of data products. This is different from access to the underlying data, which is managed by permissions in the data warehouse. For use cases where even metadata about a reusable data asset is sensitive, we are considering an optional extension of `protected` models (https://github.com/dbt-labs/dbt-core/issues/9340).

For more questions, see the FAQs on dbt Mesh: https://docs.getdbt.com/best-practices/how-we-mesh/mesh-4-faqs